packages/encrypt-upload-client/package.json

```diff
@@ -37,21 +37,9 @@
       "import": "./dist/index.js",
       "require": "./dist/index.js"
     },
-    "./factories.node": {
-      "import": "./dist/crypto/factories.node.js",
-      "require": "./dist/crypto/factories.node.js"
-    },
-    "./factories.browser": {
-      "import": "./dist/crypto/factories.browser.js",
-      "require": "./dist/crypto/factories.browser.js"
-    },
-    "./node": {
-      "import": "./dist/crypto/symmetric/node-aes-cbc-crypto.js",
-      "require": "./dist/crypto/symmetric/node-aes-cbc-crypto.js"
-    },
-    "./browser": {
-      "import": "./dist/crypto/symmetric/generic-aes-ctr-streaming-crypto.js",
-      "require": "./dist/crypto/symmetric/generic-aes-ctr-streaming-crypto.js"
+    "./factories": {
```

Comment:
I guess you will need to verify if the Console App (encryption/decryption) still works. Mainly because these factories were put into different files to be able to build and run the Console App (decryption/encryption using browser lib). If I remember correctly, the Lit lib doesn't support that and was causing several issues.